server/__db/seed-db.js

```diff
@@ -0,0 +1,14 @@
+const { Pool } = require("pg");
```

Comment:
Alternatively as a shell script, I _think_ you can do this:

```sh
NODE_ENV == production
if [[ -n "$DATABASE_CONNECTION ]]; then
  psql $DATABASE_CONNECTION -f ../__db/db-config
else
  psql -U postgres -f ../__db/db-config
fi
```

Reply:
wouldn't shell scripts be a problem for the few on windows?

Reply:
Depends if they've got bash enabled